### PR TITLE
Bug/histo

### PR DESCRIPTION
**Pull request by rnugent3**

### CODE REVIEW

**Comment by rnugent3** [comment]
testing flow and stage uncertainty got hooked into this branch ... not sure how ... but I think it is worth keeping that "test" class for the time being

File: fda-model-test/integrationtests/BootstrapToPairedDataShould.cs [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        //without all of the simulation stuff getting in the way 
+        public IPairedData BootstrapToPairedData(IProvideRandomNumbers randomProvider, ContinuousDistribution continuousDistribution, int ordinates)
+        {
+            double[] samples = randomProvider.NextRandomSequence(continuousDistribution.SampleSize);
+            IDistribution bootstrap = continuousDistribution.Sample(samples);
+            //for (int i = 0; i < dist.SampleSize; i++) samples[i] = Math.Log10(dist.InverseCDF(samples[i]));
+            //ISampleStatistics ss = new SampleStatistics(samples);
+            double[] x = new double[ordinates];
+            double[] y = new double[ordinates];
+            //double skewdividedbysix = ss.Skewness / 6.0;
```
Comment (codymccoy) [inline fda-model-test/integrationtests/BootstrapToPairedDataShould.cs:33]: Dead code, but this is a unit test, so not that important.

File: fda-model/compute/ImpactAreaScenarioSimulation.cs [hunk 2]
```diff
@@ -864,6 +864,7 @@ public SimulationBuilder withStageDamages(List<UncertainPairedData> uncertainPai
                 foreach (UncertainPairedData uncertainPairedData in _sim._damage_category_stage_damage)
                 {
                     _sim.AddSinglePropertyRule(uncertainPairedData.CurveMetaData.DamageCategory + " stage damages", new Rule(() => { uncertainPairedData.Validate(); return !uncertainPairedData.HasErrors; }, uncertainPairedData.GetErrors().ToString()));
+                    _sim.AddSinglePropertyRule("positive-damage", new Rule(() => { return uncertainPairedData.Yvals[uncertainPairedData.Yvals.Length - 1].InverseCDF(0.5) > 0; }, "Stage-damage reflects 0 damage for highest stage", HEC.MVVMFramework.Base.Enumerations.ErrorLevel.Severe));
```
Comment (codymccoy) [inline fda-model/compute/ImpactAreaScenarioSimulation.cs:867]: This is a property rule. Does that mean there is a property named "positive-damage". Properties should be named like "PositiveDamage".

File: fda-model-test/integrationtests/BootstrapToPairedDataShould.cs (hunk 1 above)
Comment (rnugent3) [reply fda-model-test/integrationtests/BootstrapToPairedDataShould.cs]: fixed

File: fda-model/compute/ImpactAreaScenarioSimulation.cs (hunk 2 above)
Comment (rnugent3) [reply fda-model/compute/ImpactAreaScenarioSimulation.cs]: fixed

**Pull request merged**